src/index.js

```diff
@@ -1,51 +1,50 @@
-var hljs = require('highlight.js');
-var React = require('react');
-var ReactDOM = require('react-dom');
-
-var Highlight = React.createClass({
-  getDefaultProps: function () {
-    return {
-      innerHTML: false,
-      className: null,
-      element: null,
-    };
-  },
-  componentDidMount: function () {
-    this.highlightCode();
-  },
-  componentDidUpdate: function () {
-    this.highlightCode();
-  },
-  highlightCode: function () {
-    var domNode = ReactDOM.findDOMNode(this);
-    var nodes = domNode.querySelectorAll('pre code');
-    if (nodes.length > 0) {
-      for (var i = 0; i < nodes.length; i=i+1) {
-        hljs.highlightBlock(nodes[i]);
-      }
+import hljs from 'highlight.js';
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class Highlight extends React.Component {
+    componentDidMount() {
+        this.highlightCode();
+    }
+
+    componentDidUpdate() {
+        this.highlightCode();
+    }
+
+    highlightCode() {
+        const domNode = ReactDOM.findDOMNode(this);
+        const nodes = domNode.querySelectorAll('pre code');
+
+        let i;
+        for (i = 0; i < nodes.length; i++) {
+            hljs.highlightBlock(nodes[i]);
+        }
     }
-  },
-  render: function () {
-    var Element = this.props.element ? React.DOM[this.props.element]: null;
-
-    if (this.props.innerHTML) {
-      if (!Element) {
-        Element = React.DOM.div
-      }
-      return Element({
-        dangerouslySetInnerHTML: {__html: this.props.children},
-        className: this.props.className || null
-      }, null);
-    } else {
-      if (Element) {
-        return Element({
-          className:this.props.className
-        }, this.props.children);
-      } else {
-        return <pre><code className={this.props.className}>{this.props.children}</code></pre>;
-      }
+
+    render() {
+        const {children, className, element, innerHTML} = this.props;
+        let Element = element ? React.DOM[element] : null;
+
+        if (innerHTML) {
+            if (!Element) {
+                Element = React.DOM.div
+            }
+
+            return Element({dangerouslySetInnerHTML: {__html: children}, className: className || null}, null);
+        } else {
+            if (Element) {
+                return Element({className}, children);
+            } else {
+                return <pre><code className={className}>{children}</code></pre>;
+            }
+        }
     }
-  }
-});
+}
+
+Highlight.defaultProps = {
```

Comment:
The recommanded way to use the `prop-types` package is like I did it and like you can see in the [docs](https://github.com/reactjs/prop-types#usage) :-)